src/plone/restapi/tests/test_documentation.py

```diff
@@ -23,8 +23,10 @@
 from plone.namedfile.file import NamedBlobImage
 from plone.registry.interfaces import IRegistry
 from plone.restapi.testing import PAM_INSTALLED
-from plone.restapi.testing import PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
-from plone.restapi.testing import PLONE_RESTAPI_DX_PAM_FUNCTIONAL_TESTING
+from plone.restapi.testing import (
+    PLONE_RESTAPI_DX_PAM_FUNCTIONAL_TESTING_FREEZETIME,
+    PLONE_RESTAPI_DX_FUNCTIONAL_TESTING_FREEZETIME
```

Comment:
Could you please split these imports onto seperate lines? That allows people to quickly keep imports sorted by just sorting lines.

Reply:
yep - I had to make them two-line because of the line length requirement.

I had this, but thought it ugly:

```
from plone.restapi.testing import (
    PLONE_RESTAPI_DX_FUNCTIONAL_TESTING_FREEZETIME
)
from plone.restapi.testing import (
    PLONE_RESTAPI_DX_PAM_FUNCTIONAL_TESTING_FREEZETIME
)
```

Reply:
btw I use isort - it handles the sorting for me ... :)

Reply:
and I added the standard plone .isort config in https://github.com/plone/plone.restapi/pull/537
makes things easier...

Reply:
You can just slap a `# noqa` at the end of the import. [See here](https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/services/auth/login.py#L7) for example. Having a as-stupid-as-they-get sort-tool being able to get the job done instead of requiring isort is preferable :)

Reply:
yep I know `# noqa`  - added.